client/lib/analytics/utils/use-do-not-sell.ts

```diff
@@ -33,6 +34,15 @@ export default () => {
 		( isActive: boolean ) => {
 			// isActive = true means user has opted out of "advertising" tracking
 			const prefs = setTrackingPrefs( { ok: true, buckets: { advertising: ! isActive } } );
+
+			if ( ! isActive ) {
```

Comment:
```suggestion
			if ( isActive ) {
```
Correct me if I am wrong, but based on the name `a8c_ccpa_optout` and the description of this PR, we want this event to trigger when somebody "opt's out" by turning **on** the "Do Not Sell or Share My Data" toggle correct?

If so, I think this change should be made, I am only getting the tracks even triggering when I turn the toggle off